.github/actions/common-versioning/action.yml

```
@@ -68,30 +80,21 @@ runs:
             exit 1
           fi
           echo "Running in release mode with version_type: ${{ inputs.version_type }}"
-          chmod +x ./scripts/version.sh
           
-          echo "Current version from version.sh get:"
-          CURRENT_VERSION=$(./scripts/version.sh get)
-          echo "$CURRENT_VERSION"
+          echo "Current version: ${CURRENT_VERSION}"
           
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)